pal/src/sslClient_arduino.cpp

```diff
@@ -15,6 +15,10 @@ static BearSSL::X509List cert(certificates);
 #include "WiFi.h"
 #include "WiFiClientSecure.h"
 static WiFiClientSecure sslClient; // for ESP32
+#elif WIO_TERMINAL
```

Comment:
The Wio Terminal WiFi header file is WiFi.h, and the SSL client comes from WiFiClientSecure.h.